setup.py

```diff
@@ -1,18 +1,15 @@
+import os
 import pathlib
-import setuptools
-import subprocess
-from setuptools import setup
-from distutils.extension import Extension
 import sys
 
+import setuptools
+from setuptools import Extension, setup
+from setuptools.command.build_ext import build_ext as _build_ext
+
 from fugashi_util import check_libmecab
 
 # get the build parameters
-if sys.argv[1] == "sdist":
-    # hack for automated builds
-    output, data_files = [], []
-else:
-    output, data_files = check_libmecab()
+output, dll_files = check_libmecab()
```

Comment:
I forget why this was here. Assuming this doesn't break anything on Linux, this is a good catch.